config/index.js

```diff
@@ -796,6 +796,18 @@ var conf = convict({
       default: 30,
       format: 'nat',
       env: 'TOTP_STEP_SIZE'
+    },
+    recoveryCodes: {
+      codeLength: {
+        doc: 'The length of a recovery code',
+        default: 8,
+        env: 'RECOVERY_CODE_LENGTH'
+      },
+      codeCount: {
```

Comment:
No biggie, but as these are already nested inside a `recoveryCodes` parent node, seems like they could just be `count` and `length` without losing clarity.